modules/core/exported/expected_keepers.go

```diff
@@ -1,4 +1,4 @@
-package types
+package exported
```

Comment:
nit:

@colin-axner wdyt about putting this into `exported/channel.go` instead of a new `expected_keepers.go` file?

Reply:
I have no preference. That makes sense to me since the scoped keeper is for channel/port capabilities

Reply:
My concern with moving this to `channel.go` would be, `CapabilityKeeper` which is `ScopedKeeper` inferface has nothing to do with `channel/port`, and it is the other way around, channel/port uses this interface.

Again since this will be anyways be exposed as the `exported` package, i will be ok moving it, reducing the files.